worldengine/draw.py

```diff
@@ -367,11 +372,13 @@ def draw_grayscale_heightmap(world, target):
 
 
 def draw_satellite(world, target):
-    ''' This draws a "satellit map" - a view of the generated planet as it may look from space '''
+    ''' This draws a "satellite map" - a view of the generated planet as it may look from space '''
```

Comment:
That would have be done for all drawing-functions - so maybe up at the top of the file?
Within this function:
`target` has to have a method `set_pixel()` and the pixels need to be accessible via `[y, x]`.